x-pack/solutions/security/plugins/security_solution/public/siem_migrations/common/components/migration_steps/configs.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { MigrationSource } from '../../types';
+import { DataInputStepId } from '../../../rules/components/data_input_flyout/steps/constants';
+import { RulesDataInput } from '../../../rules/components/data_input_flyout/steps/rules/rules_data_input';
+import { MacrosDataInput } from '../../../rules/components/data_input_flyout/steps/macros/macros_data_input';
+import { LookupsDataInput } from '../../../rules/components/data_input_flyout/steps/lookups/lookups_data_input';
+import type { QradarMigrationSteps, SplunkMigrationSteps } from '../migration_source_step/types';
+
+const SPLUNK_MIGRATION_STEPS: SplunkMigrationSteps = [
```

Comment:
This whole migration_steps folder is related to `rules` only so it should be moved to rules folder and types can go in specific folders for vendors. For example `rules/splunk` & `rules/qradar`